obspy/io/nordic/ellipse.py

```diff
@@ -0,0 +1,329 @@
+"""
+Routines for error ellipses
+ - Calculate an ellipse from covariance matrix
+ - See if a point is inside an ellipse
+ - Calculate the angle subtended by an ellipse (for back-azimuth uncert)
+ - Plot an ellipse
+
+Still missing:
+ - ellipsoids (3D ellipses)
+ - to_uncert()  (convert ellipse to covariance matrice, needed to
+                 convert QuakeML to nordic, for example)
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import copy
+
+eps = np.finfo(float).eps
+
+
+class _ellipse:
+    def __init__(self, a, b, theta=0, center=(0, 0)):
+        """Defines an ellipse
+
+        The ellipse is assumed to be centered at zero with its semi-major axis
+        axis aligned along the NORTH axis (geographic standard, not math
+        standard!) unless orientation and/or center are set otherwise
+        You can think of it as theta (geographic angle) = 90-phi (math angle)
+
+        Inputs:
```

Comment:
Can you format the docs in the same way as over obspy files? We should also add this file to the docs and check the docs for this.

Reply:
I formatted the docs